external-crates/move/tools/move-package/tests/test_sources/resolvers/successful.sh

```diff
@@ -18,6 +18,8 @@ EOF
 cat <<EOF
 [move]
 version = 0
+manifest_digest = "0x42"
```

Comment:
It looks like the format elsewhere does not include the leading "0x" -- does anything go wrong if it's included?

Reply:
Changed all of these (in all places requested)!